assets/app/features/upgrade/controllers/upgrade-database-configuration.controller.js

```diff
@@ -41,7 +46,13 @@
         vm.createServer = createServer;
         vm.connectServer = connectServer;
 
+        activate();
+
         // functions
+        function activate() {
```

Comment:
I'm not going to document `activate()` function and the rest is out of scope of this PR. If you want to do some general "catching up on the docs" please create a card for that yourself.